packages/ndla-ui/src/ResourceGroup/ResourceList.jsx

```diff
@@ -28,25 +28,48 @@ const Resource = ({
   showAdditionalResources,
   id,
   contentTypeDescription,
+  youAreHere,
 }) => {
   const hidden = resource.additional ? !showAdditionalResources : false;
+  // Designmanual is in iframe, test to get location
+  const currentPage = window.frameElement
+    ? document.referrer.includes(resource.contentUri)
+    : window.location.pathname.includes(resource.contentUri);
+
+  console.log(currentPage, resource.name);
+
+  const linkContent = (
+    <Fragment>
```

Comment:
Det skal være mulig å bruke <> og </> for Fragments nå: https://reactjs.org/docs/fragments.html#short-syntax

Reply:
Ok, da bruker vi den framover:)